index.html

```diff
@@ -108,7 +108,7 @@ <h1 class="h4 offset-top">{%- include utils/titleize.html word=collection -%}</h
             <h2 class="h5 mb-3 offset-top" id="{%- include utils/idify.html text=id -%}">{% include utils/request_method.html request_method=request_method %} {{ path[0] }}</h2>
 
             {%- if description -%}
-              <p>{{- description | newline_to_br -}}</p>
+              <p>{{- description | markdownify -}}</p>
```

Comment:
Change `<p>` to `<div>`

Reply:
[Done](https://github.com/robertlove/jekyll-openapi/compare/ef3938c60162179b98d344d7d399305503da7d0f..39678c89071c2531c37b76b5730126c8a2d02236).